docs/sources/datasources/azure-monitor/query-editor/index.md

```diff
@@ -283,6 +287,34 @@ To help you write queries, you can use several Grafana macros in the `where` cla
 | `$__escapeMulti($myVar)`        | Escapes illegal characters from multi-value template variables.<br/>If `$myVar` has the values `'\\grafana-vm\Network(eth0)\Total','\\hello!'` as a string, this expands it to `@'\\grafana-vm\Network(eth0)\Total', @'\\hello!'`.<br>If you use single-value variables, escape the variable inline instead: `@'\$myVar'`.                                                                                                                                                                        |
 | `$__contains(colName, $myVar)`  | Expands multi-value template variables.<br/>If `$myVar` has the value `'value1','value2'`, this expands it to `colName in ('value1','value2')`.<br/>If using the `All` option, then check the `Include All Option` checkbox and in the `Custom all value` field type in the following value: `all`.<br/>If `$myVar` has value `all`, this instead expands to `1 == 1`.<br/>For template variables with many options, this avoids building a large "where..in" clause, which improves performance. |
 
+## Query Application Insights Traces
+
+[Azure Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net) is a service that provides application performance monitoring (APM) features. Application insights can be used to collect metrics, telemetry, and trace logging data.
+
+Application Insights stores trace data in an underlying Log Analytics workspace and traces can be [extended](https://learn.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics) to provide additional supporting information as required by the application developer.
+
+### Create a Traces query
+
+**To create a Traces query:**
+
+1. In a Grafana panel, select the **Azure Monitor** data source.
+2. Select the **Traces** service.
+3. Select a resource to query. Multiple resources can be selected as long as they are of the same type.
+
+   > **Note:** This query type only supports Application Insights resources.
```

Comment:
Also, for admonitions like the note below, we're using [shortcodes like this](https://grafana.com/docs/writers-toolkit/writing-guide/shortcodes/#admonition-shortcode): 

```suggestion
   {{% admonition type="note" %}}
   This query type only supports Application Insights resources.
   {{% /admonition %}}
```